networks/remote/ansible/roles/setup-validators/files/nodef.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=nodef
+Requires=network-online.target ee.service
+After=network-online.target
+
+[Service]
+Restart=on-failure
+User=nodef
+Group=nodef
+PermissionsStartOnly=true
+ExecStart=/usr/bin/nodef start > /home/nodef/.nodef/nodef.log
```

Comment:
does redirection works? does or not, it doesn't matter, log redirection is redundant here.

Reply:
right, It's just tried in the past. It was accidentally included in the commit. remove now.